src/main/java/org/squiddev/cobalt/function/LuaInterpreter.java

```diff
@@ -148,7 +156,14 @@ private static DebugFrame setupCall(LuaState state, LuaInterpretedFunction funct
 		return di;
 	}
 
-	static Varargs execute(final LuaState state, DebugFrame di, LuaInterpretedFunction function) throws LuaError, UnwindThrowable {
+	static Varargs execute(LuaState state, DebugFrame di, LuaInterpretedFunction function) throws LuaError, UnwindThrowable {
+		if (true)
```

Comment:
Hrmr, I assume this should be doing something different. Are we doing to switch over to the `partialEval` after a certain number of calls, or just ditch the interpreter entirely?

Reply:
I don't think there's much use in trying to be clever about which parts of a function to interpret and which parts to partially evaluate. I keep this `if` around to quickly test the performance of `partialEval()` vs `interpret()` and to verify that the two approaches still differ solely in these two functions (i.e. edits of other methods preserve semantics).

I'll remove the `if` eventually. Would you like to keep the `interpret()` method?

Reply:
Hrmr. So the interpreter has a significantly lower memory cost than `partialEval` - so it would be good to have some sort of enhancement capability.

However, getting the two to play well together seems painful. Ideally you'd want to be able to seamlessly call between interpreted and `partialEval` functions, and I'm not sure how to do that quite yet.